queries/subsquid/bsx/offerListUser.graphql

```diff
@@ -0,0 +1,15 @@
+query offerListUser($id: String!) {
+  offers(orderBy: createdAt_DESC, where: { caller_eq: $id }) {
+    blockNumber
+    caller
+    createdAt
+    id
+    price
+    status
+    updatedAt
+    nft {
```

Comment:
it should limit for burned nft

Reply:
but doesnt it mean that it should show up on burned nfts?

Reply:
cc @petersopko

Reply:
we want to show burned NFTs so user can withdraw his offers